### PR TITLE
Task 10

### PR DESCRIPTION
**Pull request by AnnaSovpel**

### CODE REVIEW

File: task_10/style.css [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    display: none;
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    background-color: orange;
+    height: 155px;
+    }
+#okno:target {
```
Comment (Killerkip10) [inline task_10/style.css]: okno?))))))))

Good English))))

File: task_10/style.css [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+    font-size: 15px;
+    text-align: center;
+    }
+}
+@media (max-width: 465px){
+.header_section{
+    display: none;
+    }
+}
+@media (max-width: 465px){
```
Comment (Killerkip10) [inline task_10/style.css]: Why do you use the same @media(max-width: 465px)?
You can set all this styles in one
@media(max-width: 465px) {
.big_orange{
    margin-top: 54px;
    }
.menu_texst{
    margin-top: 20px;
    font-size: 15px;
    }
}

**Review by Killerkip10 — COMMENTED** [review]
## Task 10
### Total 100 points

File: task_10/style.css (hunk 1 above)
Comment (AnnaSovpel) [reply task_10/style.css]: hahah, done!)

File: task_10/style.css (hunk 2 above)
Comment (AnnaSovpel) [reply task_10/style.css]: Done.

**Pull request merged**